protocol/daemons/server/types/health_monitor.go

```diff
@@ -64,13 +64,29 @@ func (ms *healthMonitorMutableState) Stop() {
 	ms.stopped = true
 }
 
-// RegisterHealthChecker registers a new health checker for a health checkable with the health monitor. The health
+// RegisterHealthChecker registers a new health checker for a health-checkable with the health monitor. The health
 // checker is lazily created using the provided function if needed. This method is synchronized. It returns an error if
 // the service was already registered.
 func (ms *healthMonitorMutableState) RegisterHealthChecker(
 	checkable types.HealthCheckable,
 	lazyHealthCheckerCreator func() *healthChecker,
 ) error {
+	stopService := false
```

Comment:
hmm, is this change intended to be part of this PR? thought this PR was strictly adding the flags.

Reply:
Ah, looks like I mixed up some changes I meant to add to the other health monitoring PR.